README.md

```diff
@@ -1105,6 +1105,35 @@ Custom link href for the "Unlock Account" link. For this link to display, `featu
 
 Array of custom link objects `{text, href, target}` that will be added after the "Help" link. The `target` of the link is optional.
 
+#### hCaptcha options
+
+Set the following config options to customize `hCaptcha` script URI:
+
+```javascript
+// An example that uses cn1 host
+hcaptcha: {
+  scriptSource: 'https://cn1.hcaptcha.com/1/api.js',
+  scriptParams: {
+    apihost: 'https://cn1.hcaptcha.com',
+    endpoint: 'https://cn1.hcaptcha.com',
+    assethost: 'https://assets-cn1.hcaptcha.com',
+    imghost: 'https://imgs-cn1.hcaptcha.com',
+    reportapi: 'https://reportapi-cn1.hcaptcha.com',
+  }
+},
+```
+
+#### reCAPTCHA options
+
+Set the following config options to customize `reCAPTCHA` script URI:
+
+```javascript
+// An example that uses recaptcha.net
+recaptcha: {
+  scriptSource: 'https://recaptcha.net/recaptcha/api.js'
+},
+```
+
```

Comment:
Is this still accurate based on your comment that it would have no effect in widget options?

https://github.com/okta/okta-signin-widget/pull/3574/files#r1532321526

Reply:
Maybe provide instructions for SIW G2 vs G3 or set `window.recaptchaOptions={useRecaptchaNet: true};` in the widget so the API's match?

Reply:
Added setting `window.recaptchaOptions`
https://github.com/okta/okta-signin-widget/pull/3574/commits/275809ff310373e8f22ba588eb928d2c7bd9b221#diff-05bcf63656d06159b0010b0be5ad378a5358c254c95ee68222835e79d1211abdR57